weldx/asdf/schemas/weldx.bam.de/weldx/core/file-1.0.0.yaml

```
@@ -0,0 +1,84 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/weldx/core/file-1.0.0"
+tag: "tag:weldx.bam.de:weldx/core/file-1.0.0"
+
+title: |
+  Schema for a file.
+description: |
+  This schema describes a file by compiling all its meta data. Optionally, the whole file can also be stored in the
+  binary block of an asdf file.
+
+examples:
+  -
+    - A simple DHU-Groove
+    - |
+        !<tag:weldx.bam.de:weldx/core/file-1.0.0>
+          filename: file.pdf
+          suffix: .pdf
+          hostname: NB1459
+          directory: C:/Users/vhirtham/PycharmProjects/bam/custom_scripts/external_files
+          size: 90571
+          created: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2020-12-09T12:51:20.653744500'}
+          modified: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2020-12-10T13:14:17.362481500'}
+          content_hash: {algorithm: SHA-256, value: 5a6270ea5e2662c6489ee9e9c2201645e1b3cdadf0e3a621cca213a29ff4ae32}
+
+type: object
+properties:
+  filename:
+    description: |
+      The name of the file including the suffix.
+    type: string
+  suffix:
+    description: |
+      The suffix of the file.
+    type: string
+  size:
+    description: |
+      The files size in bytes.
+    type: number
+  created:
+    description: |
+      The timestamp when the file was created.
+    tag: "tag:weldx.bam.de:weldx/time/timestamp-1.0.0"
+  modified:
+    description: |
+      The timestamp when the file was modified last.
+    tag: "tag:weldx.bam.de:weldx/time/timestamp-1.0.0"
+  hostname:
+    description: |
+      The name of the files host machine.
+    type: string
+  directory:
+    description: |
+      The directory of the file on the specified host system.
+    type: string
```

```suggestion
  hostname:
    description: |
      The name of the host machine accessing the file.
    type: string
  directory:
    description: |
      The directory of the file as seen from the host machine.
    type: string
```
Is this how it works?